Source/WebCore/css/CSSShadowValue.cpp

```diff
@@ -41,8 +41,11 @@ String CSSShadowValue::customCSSText() const
 {
     StringBuilder text;
 
-    if (color)
+    if (color) {
+        if (!text.isEmpty())
+            text.append(' ');
         text.append(color->cssText());
+    }
```

Comment:
I'm quite confused at how this fixes the bug. Or why this is needed in the first place. `StringBuilder text;` will always be initialized empty, so there is no way `!text.isEmpty()` will be true.